conf/environments.js

```diff
@@ -102,6 +102,11 @@ module.exports = {
             ecmaVersion: 6
         }
     },
+    es7: {
```

Comment:
We didn't before, but we do now - the `es7` environment needs to include `globals.es6`. I added a test in `no-undef` to cover this.